### PR TITLE
{bio}[foss/2020b] WhatsHap v1.1 w/ Python 3.8.6

### PR DESCRIPTION
**Pull request by arielzn**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node378.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/801787cf81527fd7e4c1d8b6ef5e7080 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node115.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/18ccc067263223a7ddf9e4c7e5be41f2 for a full test report.

File: easybuild/easyconfigs/w/WhatsHap/WhatsHap-1.1-foss-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    ('SciPy-bundle', '2020.11'),
+    ('Biopython', '1.78'),
+    ('Yasm', '1.3.0'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('pyfaidx', '0.5.9.5', {
```
Comment (boegel) [inline easybuild/easyconfigs/w/WhatsHap/WhatsHap-1.1-foss-2020b.eb:28]: @arielzn We have a standalone easyconfig for `pyfaidx` already that is compatible with `foss/2020b`, so please make this a proper dependency

**Review by boegel — DISMISSED** [review]
@arielzn It seems like WhatsHap provides a `whatshap` command, so we should check this via both `sanity_check_paths` and `sanity_check_commands` (using `whatshap --help` for example) to ensure it's functional.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
node377.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/6babbd8f6850f14ce0f35ebbe33dd777 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
node114.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/a2f57c0d4c8d9ddc8e361064e0fcf3b9 for a full test report.

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Going in, thanks @arielzn !

**Pull request merged**